### DISCUSSION TITLE
Projects beta: automate issues ingestion

### DISCUSSION THREAD

**Discussion by pragmanomos** [community/community#12391] [Projects and Issues]
It would be nice to have a way to automatically "ingest" every new issue from a certain project (perhaps based on a query, such ass all the issues from project X, with label Y added after date Z), without having to insert them one by one.

**Comment by StephenWithPH** [comment]
At a minimum, it would be nice to have similar behavior as non-Beta projects, where the user can link multiple repositories to a given project:

![image](https://user-images.githubusercontent.com/2990492/167732192-4a6d7d74-e032-47db-b7c7-1fe814d133c8.png)

**Comment by amacneil** [comment]
+1 would be great to have a way to pull all issues from a given project into the project board
